spec/foundation_rails_helper/flash_helper_spec.rb

```diff
@@ -1,21 +1,16 @@
-# encoding: utf-8
-
 require "spec_helper"
 
 describe FoundationRailsHelper::FlashHelper do
   include ActionView::Context if defined?(ActionView::Context)
-  include ActionView::Helpers::UrlHelper
-  include ActionView::Helpers::TagHelper
-  include ActionView::Helpers::TextHelper
```

Comment:
They must have at some point.  I don't see a need to include them anymore, so I removed what I could.